<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidgetPicInPicbar</class>
 <widget class="QWidget" name="QWidgetPicInPicbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_inPic1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 161, 161);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_inPic4" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 161, 161);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_inPic2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 161, 161);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_inPic3" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 161, 161);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
